pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/ast/jdkversiontests/java15p/PatternMatchingInstanceof.txt

```diff
@@ -23,10 +23,10 @@
          |     |  +- InfixExpression[@CompileTimeConstant = "false", @Operator = "instanceof", @ParenthesisDepth = "0", @Parenthesized = "false"]
          |     |  |  +- VariableAccess[@AccessType = "READ", @CompileTimeConstant = "false", @Image = "obj", @Name = "obj", @ParenthesisDepth = "0", @Parenthesized = "false"]
          |     |  |  +- PatternExpression[@CompileTimeConstant = "false", @ParenthesisDepth = "0", @Parenthesized = "false"]
-         |     |  |     +- TypeTestPattern[]
+         |     |  |     +- TypePattern[@EffectiveVisibility = "package", @Visibility = "package"]
+         |     |  |        +- ModifierList[@EffectiveModifiers = "{}", @ExplicitModifiers = "{}"]
```

Comment:
I like that you made this more regular by moving the ModifierList here!

Reply:
Btw. your IntelliJ JavaCC plugin (https://github.com/oowekyala/intellij-javacc) is fantastic!